docs/gathered-data.md

```diff
@@ -1063,6 +1063,36 @@ Collects `MachineSet` information.
 None
 
 
+## MonitoringPVs
+
+Collects Persistent Volumes from openshift-monitoring namespace
+which matches with ConfigMap configuration yaml
+
+### API Reference
+- https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/configmap.go
+- https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/persistentvolume.go
+
+### Sample data
+- [docs/insights-archive-sample/config/persistentvolumes/monitoring-persistent-volume.json](./insights-archive-sample/config/persistentvolumes/monitoring-persistent-volume.json)
+
+### Location in archive
+- `config/persistentvolumes/{persistent_volume_name}.json`
+
+### Config ID
+`clusterconfig/monitoring_persistent_volumes`
+
+### Released version
+- 4.14
+
+### Backported versions (tentative)
+- +4.13.0
+- +4.12.6
+- +4.11.30
```

Comment:
You can't predict those values. We update the docs when a corresponding z-stream version (and erratum) is released.

Reply:
I checked Release Status webpage after this [conversation on Slack](https://redhat-internal.slack.com/archives/G019KPS1S6R/p1677845076241029). What should I put here? Just the minor version? nothing at all?

Reply:
I haven't seen the discussion I guess. Sorry. Nothing for now. We update the docs when the backport is released.

Reply:
Updated documentation and created [task](https://issues.redhat.com/browse/CCXDEV-10420) to follow-up future backports.